lib/resources/crontab.rb

```diff
@@ -25,51 +25,35 @@ class Crontab < Inspec.resource(1)
       describe crontab.where { command =~ /a partial command string/ } do
         its('entries.length') { should cmp 1 }
       end
+
+      describe crontab({path: '/etc/cron.d/some_crontab'}) do
+        its('commands') { should include '/path/to/some/script' }
+      end
     "
 
     attr_reader :params
 
     include CommentParser
 
-    def initialize(user = nil)
-      @user   = user
+    def initialize(opts = {})
+      Hash[opts.map { |k, v| [k.to_sym, v] }] if opts.respond_to?(:fetch)
+      @user = opts.respond_to?(:fetch) ? opts.fetch(:user, nil) : opts
+      @path = opts.fetch(:path, nil) if opts.respond_to?(:fetch)
       @params = read_crontab
 
       return skip_resource 'The `crontab` resource is not supported on your OS.' unless inspec.os.unix?
```

Comment:
While you're in here, can you change this to raise an exception?

```ruby
raise Inspec::Exceptions::ResourceSkipped, 'The `crontab` resource is not supported on your OS.' unless inspec.os.unix?
```

And we should also add an exception if the user didn't supply us enough information (i.e. they gave us a hash that contains neither user or path):

```ruby
raise Inspec::Exceptions::ResourceFailed, "A user or path must be supplied" if @user.nil? && @path.nil?
```